po/es.po

```diff
@@ -11333,15 +11335,15 @@ msgstr "proporción automática"
 
 #: ../src/iop/vignette.c:1153
 msgid "8-bit output"
-msgstr "salida 8-bit"
+msgstr "salida de 8 bits"
 
 #: ../src/iop/vignette.c:1154
 msgid "16-bit output"
-msgstr "salida 16-bit"
+msgstr "salida de 16 bits"
 
 #: ../src/iop/vignette.c:1159
 msgid "fall-off strength"
-msgstr "fuerza del difuminado"
+msgstr "graduación del difuminado"
```

Comment:
I don't think "graduación" is wrong but I consider simpler "fuerza".
_fall-off_ can be translated as _atenuación_ but I don't think it worth the time.

Just as a curiosity, adobe translates it as _desvanecimiento_  , I don't like it.

Reply:
This was not a very simple translation. You have to use the module and then it makes perfect sense.